pycbc/types/frequencyseries.py

```diff
@@ -430,7 +430,8 @@ def save(self, path, group=None, ifo='P1'):
                 ds = f.create_dataset(key, data=self.numpy(),
                                       compression='gzip',
                                       compression_opts=9, shuffle=True)
-                ds.attrs['epoch'] = float(self.epoch)
+                if self.epoch is not None:
+                    ds.attrs['epoch'] = float(self.epoch)
```

Comment:
Is it a problem if this is *not* set. Some fiducial value would be used when reading it back in anyway. Maybe set this to that fiducial value (is it 0?) if this is None?

Reply:
I think it *should not* be a problem in principle, i.e. I would prefer it to be set to `None` rather than to a random arbitrary time. I bet this is fine for some (most?) present applications. As far as `pycbc_make_skymap` is concerned, and for anything that runs in the automatic tests, this seems fine. I would not be surprised if this broke some edge cases, of course.

Note that when reading a `FrequencySeries` from a file, the epoch will be set to `None` if the file does not provide it (see the recent #3709).